src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/page/AddCaseNote.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.adoption.adoptioncase.caseworker.event.page;
+
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.CaseData;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.CaseNote;
+import uk.gov.hmcts.reform.adoption.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.reform.adoption.common.ccd.PageBuilder;
```

Comment:
Add comments at class and method level in next commit

Reply:
Added class level JavaDoc comments